src/ensembl/src/shared/components/layout/StandardAppLayout.scss

```diff
@@ -38,8 +38,9 @@ $drawerWindowWidth: 45px;
 }
 
 .mainWrapper {
-  height: calc(100% - #{$topbarHeight});
+  max-height: calc(100% - #{$topbarHeight});
   position: relative;
+  height: 100%;
```

Comment:
these two changed lines are probably unnecessary and should behave exactly the same as the deleted line.

Reply:
Reverted it